enterprise/admin.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+"""
+Django admin integration for enterprise app.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.contrib import admin
+
+from enterprise.actions import export_as_csv_action
+from enterprise.models import EnterpriseCustomer, EnterpriseCustomerHistory, EnterpriseCustomerMember
+
+
+@admin.register(EnterpriseCustomer)
+class EnterpriseCustomerAdmin(admin.ModelAdmin):
+    """
+    Django admin model for EnterpriseCustomer.
+    """
+
+    list_display = ("name", "short_code", "active", "data_sharing_policy", "logo")
+    search_fields = ("name", "short_code", "data_sharing_policy")
+
+    actions = [
+        export_as_csv_action("CSV Export", fields=["name", "active", "short_code", "data_sharing_policy", "logo"])
+    ]
+
+    class Meta(object):
+        """
+        Meta class for EnterpriseCustomer admin model.
+        """
+
+        model = EnterpriseCustomer
+
+
+@admin.register(EnterpriseCustomerHistory)
+class EnterpriseCustomerHistoryAdmin(admin.ModelAdmin):
+    """
+    Django admin model for EnterpriseCustomerHistory.
+    """
+
+    list_display = ("name", "origin", "active", "short_code", "data_sharing_policy", "logo", "created", "modified")
+    search_fields = ("name", "short_code", "data_sharing_policy", "created", "modified")
+    exclude = ("members",)
+
+    ordering = ["-created"]
+
+    class Meta(object):
+        """
+        Meta class for EnterpriseCustomerHistory admin model.
+        """
+
+        model = EnterpriseCustomerHistory
+
+    def has_add_permission(self, request):
+        """
+        Don't allow adds.
+        """
+        return False
+
+    def has_delete_permission(self, request, obj=None):
+        """
+        Don't allow deletes.
+        """
+        return False
```

Comment:
Should we add `readonly_fields = EnterpriseCustomerHistory._meta.get_all_field_names()` to make the model completely readonly?